src/main/java/org/ihtsdo/otf/transformationandtemplate/service/template/TemplateConceptSearchService.java

```diff
@@ -39,7 +41,10 @@ public class TemplateConceptSearchService {
 
 	@Autowired 
 	private LogicalTemplateParserService logicalTemplateParser;
-	
+
+	@Autowired
+	private TemplateConceptTransformService templateConceptTransformService;
```

Comment:
@QuyenLy87 TemplateConceptTransformationService is designed for templates transformations not for searching. I would prefer not to use it here.

Reply:
Hi @CoderMChu, I've re-used a few methods in that service. Anyway, could you review the PR again? Thanks